src/main/telemetry/smartport.c

```diff
@@ -303,13 +557,31 @@ void handleSmartPortTelemetry(void)
         return;
     }
 
+    if(smartPortFrameReceived) {
+        smartPortFrameReceived = false;
+        // do not check the physical ID here again
+        // unless we start receiving other sensors' packets
+        smartPortFrame_t* frame = (smartPortFrame_t*)smartPortRxBuffer;
```

Comment:
Why? All members in structure are correctly aligned ...

but `attribute((packed))` solves this problem (gcc assumes unaligned access then)  ...